mongomock/collection.py

```diff
@@ -518,7 +518,7 @@ def save(self, to_save, manipulate = True, safe = False, **kwargs):
                         manipulate, safe, _check_keys = True, **kwargs)
             return to_save.get("_id", None)
 
-    def remove(self, spec_or_id = None, search_filter = None):
+    def remove(self, spec_or_id = None, search_filter = None, **kwargs):
```

Comment:
Can you please rename to write_concern_kwargs to make it clear that's what it's intended for?